src/api/namespaces/atomicassets/filler.ts

```diff
@@ -1,12 +1,12 @@
-import { HTTPServer } from '../../server';
+import { DB, HTTPServer } from '../../server';
 
-export type FillerHook = (server: HTTPServer, contract: string, rows: any[]) => Promise<any[]>;
+export type FillerHook = (db: DB, contract: string, rows: any[]) => Promise<any[]>;
 
 export class AssetFiller {
     private assets: Promise<{[key: string]: any}> | null;
 
     constructor(
-        readonly server: HTTPServer,
+        readonly db: DB,
```

Comment:
Not sure whether I like this renaming. Certain APIs need access to express or especially the socket io instance to create socket APIs

Reply:
Are there APIs that are not in this repo? Both fillers that I can see only require access to the db

Reply:
It wasnt related to the filler but other APIs since you renamed it everywhere. There are rare examples like AssetApi class but only because there are not many examples implemented yet